fix typo in port number section + add note on ports for multiple dynnodes per server (#502)
ydb-platform/ydb: ydb/docs/en/core/deploy/manual/deploy-ydb-on-premises.md

* 19001, 19002: Interconnect for intra-cluster node interaction
* 8765, 8766: HTTP interface of {{ ydb-short-name }} Embedded UI.

> When hosting multiple dynamic nodes on the single server, separate ports are necessary for the GRPC, Interconnect and HTTP interface of each dynamic node within the server.

Reviewer: ```suggestion
Distinct ports are necessary for GRPC, Interconnect and HTTP interface of each dynamic node when hosting multiple dynamic nodes on a single server.
```
